website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -768,6 +782,94 @@ spec:
 ```
 {{% /alert %}}
 
+### AL2023
+
+* Your UserData may be in one of three formats: a [MIME multi part archive](https://cloudinit.readthedocs.io/en/latest/topics/format.html#mime-multi-part-archive), a NodeConfig YAML / JSON string, or a shell script.
+* Karpenter will transform your custom UserData as a MIME part, if necessary, and then merge a generated MIME part to the **beginning** of the the generated MIME multi part archive. Since `nodeadm`, the bootstrap for the AL2023 EKS optimized AMI, merges NodeConfig objects from top to bottom any fields specified in the custom UserData will override Karpenter generated fields. Take caution when overriding these fields, misconfiguration can result in Nodes being unable to connect to your cluster.
+
+#### Passed-in UserData (NodeConfig)
+
+```yaml
+apiVersion: node.eks.aws/v1alpha1
+kind: NodeConfig
+spec:
+  kubelet:
+    config:
+      maxPods: 42
+```
+
+#### Passed-in UserData (Bash)
+
+```shell
+#!/bin/bash
+echo "Hello, AL2023!"
+```
+
+#### Passed-in UserData (MIME)
+
+```shell
+MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="//"
+
+--//
+Content-Type: application/node.eks.aws
+
+apiVersion: node.eks.aws/v1alpha1
+kind: NodeConfig
+spec:
+  kubelet:
+    config:
+      maxPods: 42
+--//
+Content-Type: text/x-shellscript; charset="us-ascii"
+
+#!/bin/bash
+echo "Hello, AL2023!"
+--//
+```
+
+#### Merged UserData
```

Comment:
Considering you gave three examples above, I'm not quite sure I get why there is only one merged section in the docs

Reply:
I'd personally do: An example of what the passed in data is followed by the merged result. You can section them out and do it 3 times. FWIW, I find the AL2 example also confusing.